pkgs/data/fonts/myrica/default.nix

```diff
@@ -5,8 +5,9 @@ fetchFromGitHub {
 
   owner = "tomokuni";
   repo = "Myrica";
-  rev = "b737107723bfddd917210f979ccc32ab3eb6dc20";
-  sha256 = "187rklcibbkai6m08173ca99qn8v7xpdfdv0izpymmavj85axm12";
+  # commit does not exist on any branch on the target repository
```

Comment:
Isn't this commit found is `master` :thinking: ?

Reply:
Almost all commits are no different and no longer contain the TCC files.